doc/tutorial/statistical_inference/model_selection.rst

```diff
@@ -41,59 +41,67 @@ data in *folds* that we use for training and testing::
     >>> print(scores)
     [0.93489148580968284, 0.95659432387312182, 0.93989983305509184]
 
-.. currentmodule:: sklearn.cross_validation
+.. currentmodule:: sklearn.model_selection
 
 This is called a :class:`KFold` cross validation
 
 .. _cv_generators_tut:
 
-Cross-validation generators
-=============================
+Cross-validation classes
+========================
 
 
 
-The code above to split data in train and test sets is tedious to write.
-Scikit-learn exposes cross-validation generators to generate list
-of indices for this purpose::
+The above code, to split data into train and test sets, is tedious to write. 
```

Comment:
Commas fine, but the sentence would be better reordered or, as @vene suggests, made less presumptuous or removed!